train.py

```diff
@@ -42,7 +44,10 @@ def get_arguments():
                         help='Whether to store advanced debugging information '
                         '(execution time, memory consumption) for use with '
                         'TensorBoard.')
-    parser.add_argument('--logdir', type=str, default=LOGDIR,
+    parser.add_argument('--logdir', type=str, default=get_default_logdir(),
+                        help='Directory in which to store the logging '
+                        'information for TensorBoard.')
+    parser.add_argument('--checkpoint_dir', type=str, default=LOGDIR,
                         help='Directory in which to store the logging '
```

Comment:
It's reasonable 👍

Maybe I should add some validations:
 - Reject specifying both of `--logdir` and `--logdir_root`
 - Reject specifying both of `--logdir` and `--restore_from`  
It is a bit danger a bit because it could unexpectedly overwrite previous model in `--logdir` by new training. Maybe we should add the interactive confirmation or a flag like `-f`, `--force` to ignore the guard, but I want leave it as another issue at this time.